src/views/forms/TriggersForm/AdvancedTriggersOptions.tsx

```diff
@@ -0,0 +1,59 @@
+import styled, { AnyStyledComponent } from 'styled-components';
+
+import { STRING_KEYS } from '@/constants/localization';
+
+import { useStringGetter } from '@/hooks';
+
+import { layoutMixins } from '@/styles/layoutMixins';
+
+import { HorizontalSeparatorFiller } from '@/components/Separator';
+
+import { LimitPriceInputs } from './LimitPriceInputs';
+import { OrderSizeInput } from './OrderSizeInput';
+
+type ElementProps = {
+  symbol: string;
+  stepSizeDecimals?: number;
+  tickSizeDecimals?: number;
+};
+
+type StyleProps = {
+  className?: string;
+};
+
+export const AdvancedTriggersOptions = ({
+  symbol,
+  stepSizeDecimals,
+  tickSizeDecimals,
+  className,
+}: ElementProps & StyleProps) => {
+  const stringGetter = useStringGetter();
+
+  return (
+    <Styled.Container>
+      {
+        <Styled.Header>
+          {stringGetter({ key: STRING_KEYS.ADVANCED })}
+          {<HorizontalSeparatorFiller />}
+        </Styled.Header>
+      }
+      {/* TODO: CT-625 Update with values from abacus */}
+      {<OrderSizeInput className={className} symbol={symbol} stepSizeDecimals={stepSizeDecimals} />}
+      {<LimitPriceInputs className={className} tickSizeDecimals={tickSizeDecimals} />}
```

Comment:
Are the brackets remnants of comments or conditionals?

Reply:
ah sry - cleaned up - I think these are just remenants of me thinking and being bracket aggressive